tests/real/community.rs

```diff
@@ -729,22 +729,38 @@ async fn test_panic_in_handler() {
     let local = "test_panic_in_handler";
 
     let mut w = Worker::builder::<io::Error>()
-        .register_blocking_fn("panic", |_j| {
+        .register_blocking_fn("panic_SYNC_handler", |_j| {
             panic!("Panic inside the handler...");
         })
+        .register_fn("panic_ASYNC_handler", |j| async move {
+            // potentially going out of bounds
+            let arg1 = &j.args()[0];
+            let arg2 = &j.args()[1];
+            let _ = arg1.as_i64().unwrap() + arg2.as_i64().unwrap();
+            Ok::<(), io::Error>(())
+        })
         .connect(None)
         .await
         .unwrap();
 
-    Client::connect(None)
-        .await
-        .unwrap()
-        .enqueue(Job::builder("panic").queue(local).build())
+    let mut c = Client::connect(None).await.unwrap();
```

Comment:
The more I look at `connect(None)`, the less I like it. I wonder if we should have two methods:
```rust
fn connect() { .. }
fn connect_to(addr: &str) { .. }
```
what do you think?

Reply:
I think the suggested changes will make the bindings more ergonomic, because now even though they are already providing FAKTORY_URL into the environment, we still make them pass `None` to the the `Client::connect`.

What do you make of batching these changes with my earlier [suggestions](https://github.com/jonhoo/faktory-rs/pull/64#issuecomment-2278094341) in a follow-up "polishing" PR? Also what do you make of those suggestions?

Reply:
Yeah, I'm fine doing that change in a separate PR — this one is already growing quite large and unwieldy! I should have used reviewable from the start for it :p